src/routes/controllers/roles.controller.js

```diff
@@ -0,0 +1,84 @@
+import models from '../../db/models';
+import {
+  successResponse,
+  errorResponse,
+  getUserbyEmail,
+} from '../../utils/helpers.utils';
+import { AUTHOR, ADMIN } from '../../utils/constants';
+
+const { User } = models;
+
+/**
+ * Fetch all users and roles
+ *
+ * @param {Object} req express request object
+ * @param {Object} res express response object
+ * @returns {*} success response
+ * @throws {*} error if database error
+ */
+export async function getAll(req, res) {
```

Comment:
lets rename this controller to `getAllRoles`

Reply:
this has been renamed